cmd/opdoc/opdoc.go

```diff
@@ -166,39 +165,64 @@ func stackMarkdown(op *logic.OpSpec) string {
 	return out + "\n"
 }
 
-func opToMarkdown(out io.Writer, op *logic.OpSpec, groupDocWritten map[string]bool) (err error) {
+func opToMarkdown(out io.Writer, specs []logic.OpSpec, groupDocWritten map[string]bool) (err error) {
 	ws := ""
+	if len(specs) == 0 {
+		return nil
+	}
```

Comment:
When would this happen? Seems like you must have looked up an opcode and gotten nothing back.

Reply:
Good point, initially I think I was going to write opNames that were above the max version like the current bls ones and then just have specsbyname return nil for them